fsmonitor.c

```diff
@@ -222,6 +222,7 @@ void refresh_fsmonitor(struct index_state *istate)
 				istate->fsmonitor_last_update, &query_result);
 
 			if (query_success) {
+				strbuf_release(&last_update_token);
```

Comment:
Better `strbuf_reset()`; This avoids releasing and immediately re-acquiring memory. It's a pattern to avoid.